com.richwhitehouse.BigPEmu.metainfo.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.richwhitehouse.BigPEmu</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary=ReadMe.txt</project_license>
```

Comment:
is the binary redistributable?

Reply:
btw this should link to a URL not a random file name

Reply:
Yes, binary redistributable.
This license information is only contained within the archive, any suggestions on how to define it?

Reply:
If you can ask upstream to publish the license somehwere online on the site it'd be nice.

Otherwise just use `LicenseRef-proprietary`, a file cannot be linked like that.